management-api-for-apache-cassandra-4.1.yaml

```diff
@@ -26,14 +26,15 @@ pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/k8ssandra/management-api-for-apache-cassandra
-      expected-commit: 1822715aba5cc5e475b8ac7a7a470ba70824eac2
+      expected-commit: 30be84dcdbfd39314ab320366a08710ad80a9b92
       tag: v${{package.version}}
 
   - uses: patch
     with:
       patches: upgrade-deps.patch GHSA-5jpm-x58v-624v.patch
 
   - runs: |
+      echo "Running build..."
```

Comment:
There can be a long delay before anything is printed to the console when the build is initiated. This was giving the false impression that the patches (previous step), were causing an issue. Simple log statement which makes it clearer that the patches succeeded and we're onto the build step